config/purposes/wgs_isc_top.yml

```diff
@@ -43,6 +43,11 @@ LB Lib PCR-XP:
       id: concentrations_nm
     - name: Download Concentration (ng/ul) CSV
       id: concentrations_ngul
+Library Stock Plate: #new plate purpose for multiplexed sequencing submissions, to avoid FC
+  :asset_type: plate
+  :stock_plate: true
+  :input_plate: true
+  :presenter_class: Presenters::StockPlatePresenter
```

Comment:
I would put this in its own file. As it's shared between several pipelines, it doesn't make sense in a file for a particular pipeline.

The file that it's in has no significance, technically, as they are all concatenated together. So just for readability.